### ISSUE TITLE
Icon and Atom

### ISSUE THREAD

**Issue by cfjedimaster** [driftyco/ionic-lab-issues#12]
Ok, so this is _super_ petty, but the icon is very similar to Atom.

![image](https://cloud.githubusercontent.com/assets/393660/8762008/f8108c7a-2d31-11e5-99e6-a5e3d5b0a7de.png)

Close enough to be confusing.

**Comment by cfjedimaster** [comment]
Hmm - that pasted weird. Let me try again.

![image](https://cloud.githubusercontent.com/assets/393660/8762009/08e68eaa-2d32-11e5-9ef2-f0771a473a4e.png)

**Comment by cfjedimaster** [comment]
boggle - I don't normally have issues pasting images from Skitch. One more time.
<img width="94" alt="workdamnit" src="https://cloud.githubusercontent.com/assets/393660/8762011/1d89a784-2d32-11e5-9528-0c12ad11e6e7.png">

**Comment by bensperry** [comment]
Yeah, this current one is going to be replaced. I realized this needed to happen the other day after getting my iPhone serviced...

![image](https://cloud.githubusercontent.com/assets/519526/8763235/f583af88-2d58-11e5-9593-c35eb150ff28.png)

**Comment by mlynch** [comment]
LOL

**Comment by jbavari** [comment]
We've got a new icon in there. Will be rolled out in the next release!
